csharp/ql/test/library-tests/expressions/PrintAst.expected

```diff
@@ -825,6 +825,8 @@ expressions.cs:
 #  122|                   9: [TypeofExpr] typeof(...)
 #  122|                     0: [TypeAccess] access to type Y<,>
 #  122|                       0: [TypeMention] Y<T, U>
+#  122|                         1: [TypeMention] <unknown type>
+#  122|                         2: [TypeMention] <unknown type>
```

Comment:
Are these expected? This is extracted from `typeof(Y<,>)`

Reply:
Not sure if it is expected, but this is because `Y<T>` in `class Y<T> { ... }` and `Y<>` are considered the same (the unbound declaration). So in `typeof(Y<>)` there is an implicit type argument, which is the type parameter itself, and apparently Roslyn struggles with that and gives it kind `TypeKind.Error`.